packages/fxa-content-server/app/scripts/models/reliers/relier.js

```diff
@@ -46,6 +47,7 @@ const QUERY_PARAMETER_SCHEMA = {
   migration: Vat.string().valid(Constants.AMO_MIGRATION, Constants.SYNC11_MIGRATION),
   reset_password_confirm: Vat.boolean().renameTo('resetPasswordConfirm'),
   setting: Vat.string(),
+  style: Vat.string().valid('trailhead'),
```

Comment:
add `trailhead` to constants like we have above for migration? use it below as well?

Reply:
Good suggestion, I'll get to that on Monday.